src/migrations/20251740066033_user_info.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  await knex.schema.createTable('user_info', table => {
+    table.increments();
+    table.string('sub', 50).notNullable().comment('Subject - Identifier for the End-User at the Issuer.');
+    table.string('name', 200).nullable().comment('End-User\'s full name in displayable form including all name parts, possibly including titles and suffixes, ordered according to the End-User\'s locale and preferences.');
+    table.string('middle_name').nullable().comment('Casual name of the End-User that may or may not be the same as the given_name.');
+    table.string('given_name', 30).nullable().comment('Given name(s) or first name(s) of the End-User.');
+    table.string('family_name', 30).nullable().comment('Surname(s) or last name(s) of the End-User.');
+    table.date('birthdate').nullable().comment('End-User\'s birthday');
+    table.integer('updated_at').unsigned().nullable().comment('Time the End-User\'s information was last updated.');
```

Comment:
This should be `modified_at` for consistency with other tables.

Might also be nice to add `created_at`. Most other tables also have this.

Reply:
Ooh should it be that, or should I follow the spec? https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims:~:text=Section%C2%A05.1.1.-,updated_at,-number

Reply:
`created_at` added... always wonder how people get that exact time right :p

Reply:
It should be `modified_at`, because this server doesn't just support OIDC. It's own internal data-model should be consistent with itself. So this also uses camelCase in the API for example.

But when we need to encode `modified_at` for OpenID response bodies, it does get changed to `updated_at`. I think if I started from 0 I would probably go for `updated_at` everywhere because it seems a bit more common, but all tables right now use `modified_at`.

Reply:
So I kind of see a12n-server in the first place be consistent with itself, but then if we need to implement any standard or protocol we do convert to whatever they want.

Reply:
Oh yeah, I was wondering... when to use camelCase and when to use underscore 😕